Mathlib/Data/Set/Intervals/Basic.lean

```diff
@@ -1947,3 +1947,38 @@ instance : NoMaxOrder (Set.Iio x) :=
     exact ⟨⟨b, hb₂⟩, hb₁⟩⟩
 
 end Dense
+
+/-! ### Lemmas about intervals in subtypes
```

Comment:
IMHO, this file is already too large. What do you think about moving these lemmas to a new file?

Reply:
I've moved to another file.